tests/foreman/cli/test_satellitesync.py

```diff
@@ -68,41 +66,17 @@ def export_import_cleanup_function(target_sat, function_org):
     """Deletes export/import dirs of function org"""
     yield
     # Deletes directories created for export/import test
-    target_sat.execute(f'rm -rf {EXPORT_DIR}/{function_org.name}')
-    target_sat.execute(f'rm -rf {IMPORT_DIR}/{function_org.name}')
+    target_sat.execute(f'rm -rf {PULP_EXPORT_DIR}/{function_org.name}')
+    target_sat.execute(f'rm -rf {PULP_IMPORT_DIR}/{function_org.name}')
 
 
 @pytest.fixture(scope='function')  # perform the cleanup after each testcase of a module
 def export_import_cleanup_module(target_sat, module_org):
     """Deletes export/import dirs of module_org"""
     yield
     # Deletes directories created for export/import test
-    target_sat.execute(f'rm -rf {EXPORT_DIR}/{module_org.name}')
-    target_sat.execute(f'rm -rf {IMPORT_DIR}/{module_org.name}')
-
-
-def validate_filepath(sat_obj, org):
-    """Checks the existence of certain files in a dir"""
-    result = sat_obj.execute(
-        fr'find {EXPORT_DIR}{org.name} -type f \( -name "*.json" -o -name "*.tar.gz" \)'
-    )
-    return result.stdout
-
-
-def move_pulp_archive(sat_obj, org, export_message):
-    """
-    Moves exported archive(s) and its metadata into import directory,
-    sets ownership, returns import path
-    """
-    sat_obj.execute(f'mv {EXPORT_DIR}/{org.name} {IMPORT_DIR}')
-    sat_obj.execute(f'chown -R pulp:pulp {IMPORT_DIR}')
-
-    # removes everything before export path,
-    # replaces EXPORT_PATH by IMPORT_PATH,
-    # removes metadata filename
-    import_path = os.path.dirname(re.sub(rf'.*{EXPORT_DIR}', IMPORT_DIR, export_message))
-
-    return import_path
+    target_sat.execute(f'rm -rf {PULP_EXPORT_DIR}/{module_org.name}')
+    target_sat.execute(f'rm -rf {PULP_IMPORT_DIR}/{module_org.name}')
```

Comment:
same comment here about running both commands at once. you could even just add both paths to the rm